server/helpers/adapters_tracker.go

```diff
@@ -75,6 +78,7 @@ func (a *AdaptersTracker) DeployAdapter(ctx context.Context, adapter models.Adap
 	}
 
 	// Deploy to current platform
+	targetIP := "127.0.0.1"
```

Comment:
Also, see line 73, "AddAdapter" should be "DeployAdapter"

Reply:
fixed